Makefile

```diff
@@ -34,16 +33,16 @@ test-integration: package
 	  export COMPOSER_HOME=$(shell pwd)/build/tools/.composer && \
 	  chmod +x build/toolbox.phar && \
 	  mkdir -p ./build/tools && \
-	  build/toolbox.phar install --target-dir ./build/tools --exclude-tag exclude-php:$(PHP_VERSION) && \
-	  build/toolbox.phar test --target-dir ./build/tools --exclude-tag exclude-php:$(PHP_VERSION)
```

Comment:
This part should stay. I'd like to keep the exclude functionality. Even though currently none of the tools would be using it, I'm sure the need for it will come back.